addons/core/translations/actions/en-us.yaml

```diff
@@ -52,3 +52,4 @@ toggle-color-theme: Toggle Color Theme
 toggle-feature-edition: Toggle Feature Edition
 toggle-licensed-features: Toggle Licensed Features
 more: ', +{remainingItems} more'
+new-shell: New Shell
```

Comment:
We can keep this but just wanted to mention this translation only makes sense if we can open multiple terminals